src/renderer/components/+workloads-pods/pod-details-secrets.scss

```diff
@@ -1,7 +1,7 @@
 .PodDetailsSecrets {
-  a {
+  > * {
     display: block;
-    margin-bottom: $margin;
+    margin-bottom: var(--margin);
```

Comment:
Changing from sass to css variables in different places for sake of future theme optimization.

Reply:
I thought we do all the conversion between sass and css variables in the one theme file. Or by "future optimization" you mean that it is better if we don't do that

Reply:
I'm about removing extra step between theme itself (introduced in `json` file) and theme consumers (scss files). In general, we don't need `src/renderer/themes/theme-vars.scss` file, because we can just use plain css vars everywhere.

Reply:
Oh right, we are already loading them as css vars. So we might as well use them as such. Makes sense